### DISCUSSION TITLE
Problem with update from 3.2.3 to 4.0

### DISCUSSION THREAD

**Discussion by NoipName** [thorsten/phpMyFAQ#3292] [Q&A]
Hello,

i tried to update from 3.2.3 to 4.0.1 and i have some issue

there is a message 
![image](https://github.com/user-attachments/assets/41e4493a-b423-4735-b708-043f01eb0db0)

I run with last version php 8.3.14

i also update nginx.conf code for the lastone

If i click "Go back to admin backend" i have this error 
![image](https://github.com/user-attachments/assets/9054b53f-bc74-4914-a800-e29f5549c21c)

.htaccess was modify to use the location of phpmyfaq

Someone have the same problem ? 

Have a nice day and thank advance :D

**Comment by thorsten** [comment]
Hi,

you modified the RewriteBase? Is AllowOverride All activated?

-Thorsten

**Reply by NoipName** [reply]
> Hi,
> 
> you modified the RewriteBase? Is AllowOverride All activated?
> 
> -Thorsten

Hello thorsten,

Nope and i just add it in .htaccess and tested it, same problem :(

**Comment by thorsten** [comment]
@NoipName AllowOverride All needs to be activated, otherwise it won't work

**Reply by NoipName** [reply]
> @NoipName AllowOverride All needs to be activated, otherwise it won't work

Argument added in .htaccess
![image](https://github.com/user-attachments/assets/1f984d1c-83c2-481f-a423-cabffd5c5de6)

Nginx file:
server {
        listen 443 ssl http2;

        server_name phpmyfaq.test.com;
        access_log /var/log/nginx/phpmyfaq.access.log;
        error_log /var/log/nginx/phpmyfaq.error.log;

        ssl_certificate /etc/nginx/ssl/certs.pem;
        ssl_certificate_key /etc/nginx/ssl/certs.key;

        ssl_protocols TLSv1.2;
        ssl_prefer_server_ciphers on;
        ssl_ciphers "EECDH+ECDSA+AESGCM EECDH+aRSA+AESGCM EECDH+ECDSA+SHA384 EECDH+ECDSA+SHA256 EECDH+aRSA+SHA384 EECDH+aRSA+SHA256 EECDH+aRSA+RC4 EECDH EDH+aRSA HIGH !RC4 !aNULL
 !eNULL !LOW !3DES !MD5 !EXP !PSK !SRP !DSS";

        if ($allow_ip = no) {
        return 404;
        }

        root /srv/www/phpmyfaq;
        index       index.php index.html index.htm;


    rewrite //  /           break;
    rewrite ^/$ /index.php  last;

    # Gzip Settings
    gzip on;
    gzip_types text/plain text/css application/json application/javascript text/xml application/xml application/xml+rss text/javascript;
    gzip_proxied no-cache no-store private expired auth;
    gzip_min_length 1000;
    gzip_comp_level 6;
    gzip_vary on;
    gzip_buffers 16 8k;
    gzip_disable "MSIE [1-6]\.(?!.*SV1)";

    location ~* \.(jpg|jpeg|gif|png|webp|svg|ico|mp4|webm|mpeg|ttf|otf|woff|woff2|css|js|pdf)$ {
        expires 1y;
        add_header Cache-Control "public";
    }

    # Additional configurations for specific file types
    location ~* \.(svg|eot|ttf|woff|woff2)$ {
        add_header Access-Control-Allow-Origin "*";
    }

    # Specific cache-control for javascript files
    location ~* \.(js)$ {
        expires 1y;
        add_header Cache-Control "public, max-age=31536000, immutable";
    }

    # Specific cache-control for CSS files
    location ~* \.(css)$ {
        expires 1y;
        add_header Cache-Control "public, max-age=31536000, immutable";
    }

    # Rewrite logging, should be turned off on production
  #  rewrite_log on

    # X-Frame-Options to prevent clickjacking
    add_header X-Frame-Options SAMEORIGIN;

    location / {
        index index.php;

        if (!-f $request_filename) {
            # General pages
            rewrite ^/add-faq.html$                                 /index.php?action=add last;
            rewrite ^/add-question.html$                            /index.php?action=ask last;
            rewrite ^/show-categories.html$                         /index.php?action=show last;
            rewrite ^/(search|open-questions|help|contact|glossary|overview|login|privacy|index).html$ /index.php?action=$1 last;
            rewrite ^/(login)                                       /index.php?action=login last;

            # a solution id page
            rewrite ^/solution_id_([0-9]+).html$                    /index.php?solution_id=$1 last;

            # the bookmarks page
            rewrite ^/bookmarks.html$                               /index.php?action=bookmarks last;

            # PMF faq record page
            rewrite content/([0-9]+)/([0-9]+)/([a-z\-]+)/(.+).html$ /index.php?action=faq&cat=$1&id=$2&artlang=$3 last;

            # PMF category page with page count
            rewrite category/([0-9]+)/([0-9]+)/(.+).html$           /index.php?action=show&cat=$1&seite=$2 last;

            # PMF category page
            rewrite category/([0-9]+)/(.+).html$                    /index.php?action=show&cat=$1 last;

            # PMF news page
            rewrite news/([0-9]+)/([a-z\-]+)/(.+).html$             /index.php?action=news&newsid=$1&newslang=$2 last;

            # PMF sitemap
            rewrite sitemap/([^\/]+)/([a-z\-]+).htm(l?)$            /index.php?action=sitemap&letter=$1&lang=$2 last;

            # PMF Google sitemap
            rewrite sitemap.xml$ /sitemap.xml.php last;
            rewrite sitemap.gz$ /sitemap.xml.php?gz=1 last;
            rewrite sitemap.xml.gz$ /sitemap.xml.php?gz=1 last;

            # robots.txt
            rewrite robots.txt$ /robots.txt.php last;

            # PMF tags page with page count
            rewrite tags/([0-9]+)/([0-9]+)/(.+).htm(l?)$            /index.php?action=search&tagging_id=$1&seite=$2 last;

            # PMF tags page
            rewrite tags/([0-9]+)/([^\/]+).htm(l?)$                 /index.php?action=search&tagging_id=$1 last;

            # Authentication services
            rewrite services/webauthn(.*)                           /services/webauthn/index.php last;

            # User pages
            rewrite user/(ucp|bookmarks|request-removal|logout|register) /index.php?action=$1 last;

            # Private APIs
            rewrite api/(autocomplete|bookmark/([0-9]+)|user/data/update|user/password/update|user/request-removal|contact|voting|register|captcha|share|comment/create|faq/create
|question/create|webauthn/prepare|webauthn/register|webauthn/prepare-login|webauthn/login) /api/index.php last;

            # Setup and update pages
            rewrite setup/                                          /setup/index.php last;
            rewrite update/                                         /update/index.php last;

            # Setup APIs
            rewrite api/setup/(check|backup|update-database)        /api/index.php last;

            # Administration API
            rewrite admin/api/(.*)                                  /admin/api/index.php last;

            # Administration pages
            rewrite admin/(attachments|backup|category|comments|configuration|elasticsearch|export|glossary|group|import|instance|instances|news|password|questions|session-keep-a
live|statistics|sticky-faqs|stopwords|system|tags|update|user) /admin/front.php last;

            # REST API v3.0 and v3.1
            rewrite ^api/v3\.[01]/(.*)                              /api/index.php last;

            break;
        }
    }

    try_files $uri @php;

    location @php {
        fastcgi_pass unix:/run/php/php8.3-fpm.sock;
#        fastcgi_pass    127.0.0.1:9000;
        fastcgi_index   index.php;
        fastcgi_param   SCRIPT_FILENAME $document_root/index.php;
        include         /etc/nginx/fastcgi_params;
    }

    location ~ '/.+\.ph(p|tml)(/|$)' {
        fastcgi_pass unix:/run/php/php8.3-fpm.sock;
#        fastcgi_pass    127.0.0.1:9000;
        fastcgi_index   index.php;
        fastcgi_param   SCRIPT_FILENAME $document_root/$fastcgi_script_name;
        include         /etc/nginx/fastcgi_params;
    }

    location ~ /\.ht {
        deny all;
    }

    location ~* \.(js|css|png|jpg|jpeg|gif|ico)$ {
        expires 2d;
        add_header Cache-Control "no-store";
    }
}



Here is my code i think all is correct

**Comment by thorsten** [comment]
ah, you're using nginx, sorry!

Is the `location /` correct?
